provider/ansible.py

```diff
@@ -116,3 +107,56 @@ def store_playbook_log(self, log_dir, filename):
         with open(os.path.join(log_dir, filename), 'w') as log_file:
             log_file.write(self.get_output())
             log_file.flush()
+
+
+def check_ansible_playbook(params):
+    """
+    check if ansible-playbook exists or not.
+
+    :param params: Dictionary with the test parameters.
+    :return: True if ansible-playbook exists or be installed success, else False.
+    """
+
+    def python_install():
+        """
+        Install python ansible.
+        """
+        pip_bin = ''
+        for binary in ['pip', 'pip3', 'pip2']:
+            if process.system("which %s" % binary, ignore_status=True) == 0:
+                pip_bin = binary
```

Comment:
Need to break the loop when the pip is found.

Reply:
Good catch, updated, thanks!